prov/rxm/src/rxm.h

```diff
@@ -197,12 +197,15 @@ struct rxm_iov {
 };
 
 enum rxm_buf_pool_type {
-	RXM_BUF_POOL_RX		= 0,
-	RXM_BUF_POOL_TX_MSG	= 1,
+	RXM_BUF_POOL_START	= 0,
+	RXM_BUF_POOL_RX		= RXM_BUF_POOL_START,
+	RXM_BUF_POOL_TX_START	= 1,
+	RXM_BUF_POOL_TX_MSG	= RXM_BUF_POOL_TX_START,
 	RXM_BUF_POOL_TX_TAGGED	= 2,
 	RXM_BUF_POOL_TX_ACK	= 3,
-	RXM_BUF_POOL_TX_LMT	= 4,
-	RXM_BUF_POOL_MAX_VAL	= 5,
+	RXM_BUF_POOL_TX_END	= 4,
+	RXM_BUF_POOL_TX_LMT	= RXM_BUF_POOL_TX_END,
```

Comment:
Drop the direct assignments and let the enum increment naturally.  Swap the order of TX_LMT and TX_END, so TX_END = TX_LMT.

Btw, LMT always reads to me as 'limit' not 'large message transfer'.  We need to fix this someday.  Maybe use RNDZ for rendezvous protocol?

Reply:
> Maybe use RNDZ for rendezvous protocol?

Can I do this in a separate PR?

Reply:
> Drop the direct assignments and let the enum increment naturally. Swap the order of TX_LMT and TX_END, so TX_END = TX_LMT.

Done